.github/scripts/install_dep.sh

```diff
@@ -12,9 +12,8 @@ sudo apt install maven
 echo "Install nodeJS"
 
 sudo apt-get install curl
-curl -sL https://deb.nodesource.com/setup_13.x
-sudo -E bash -
-sudo apt-get install nodejs
+curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+sudo apt-get install -y nodejs yarn
```

Comment:
The Maven build actually installs node and yarn itself (and it controls the versions).  It is those that are used for the build.  What is this node actually used for?

Reply:
@k-wall Really? I didn't know that, actually I am wondering what is bad on my local machine in that case because I've hit several times issue when my build fails with error missing nodejs :/

Reply:
Ah I see via maven-front-end plugin, in that case I should remove this nodeJS installing all the way.